app/styles/app.css

```diff
@@ -318,4 +318,80 @@ pre {
 }
 .popover-button i{
   color:#666;
-}
\ No newline at end of file
+}
+
+
+.user-feedback{
+  position:fixed;
+  bottom:100px;
```

Comment:
A thought; is this ok as a fixed distance or is there value in using a percentage here?

Reply:
I don't think that having a percentage would change much its just the button sitting on the side of the screen

Reply:
Also it's pretty far down the page, what do we think users would do if it were at 50% or further up? @cameronblandford thoughts?

Reply:
I think having it most of the way down the page keeps it from being too distracting, and as soon as the user scrolls even a tiny bit, they'll absolutely notice it. I think it's obvious but not detracting, where it is. I wouldn't move it

Reply:
Ok cool.